src/handlers/completion.rs

```diff
@@ -241,6 +241,7 @@ async fn find_completions(
                     get_user_matches(info, ctx).await?;
 
                 items.append(&mut user_matches);
+                // Find imports aliases
```

Comment:
can you help me understand which line this comment applies too? Is it the `user_matches`?

Reply:
😬 sorry that was a planning comment I forgot to remove